<launch>
    <!-- Include this in your launch file to change port -->
    <!-- <param name="port" type="str" value="/dev/ttyTHS1"/> -->

    <rosparam file="$(find hoverboard_driver)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find hoverboard_driver)/config/controllers.yaml" command="load"/>
    <node name="hoverboard_driver" pkg="hoverboard_driver" type="hoverboard_driver" output="screen"/>
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen"
        args="hoverboard_joint_publisher hoverboard_velocity_controller" />
</launch>
